openbas-front/src/actions/mapper/mapper-actions.ts

```diff
@@ -39,3 +47,15 @@ export const testXlsFile = (importId: string, input: InjectsImportTestInput) =>
   const uri = `${XLS_MAPPER_URI}/store/${importId}`;
   return simplePostCall(uri, input);
 };
+
+export const exportMapper = (input: ExportMapperInput) => {
+  const uri = `${XLS_MAPPER_URI}/export`;
+  return simplePostCall(uri, input).then((response) => {
+    return { data: response.data, filename: response.headers['content-disposition'].split('filename=')[1] };
+  });
+};
+
+export const importMapper = (formData: FormData) => (dispatch: Dispatch) => {
```

Comment:
Why are you using a dispatch here ?

Reply:
Same as https://github.com/OpenBAS-Platform/openbas/pull/1263/files#r1698685702